README.rst

```diff
@@ -53,15 +53,15 @@ This example will get a player with a certain tag, and search for 5 clans with a
 
     async def main():
         player = await client.get_player("tag")
-        print("{0.name} has {0.trophies} trophies!".format(player))
+        print(f"{player.name} has {player.trophies} trophies!")
 
         clans = await client.search_clans(name="Best Clan Ever", limit=5)
         for clan in clans:
-            print("{0.name} ({0.tag}) has {0.member_count} members".format(clan))
+            print(f"{clan.name} ({clan.tag}) has {clan.member_count} memberswar)
```

Comment:
Not sure what happened to the end of this line, but the double quote isn't closed.